pkg/registry/pod/etcd/etcd.go

```diff
@@ -35,6 +35,7 @@ import (
 	"k8s.io/kubernetes/pkg/registry/generic"
 	etcdgeneric "k8s.io/kubernetes/pkg/registry/generic/etcd"
 	genericrest "k8s.io/kubernetes/pkg/registry/generic/rest"
+	nodeetcd "k8s.io/kubernetes/pkg/registry/node/etcd"
```

Comment:
Last step: move (or copy) the interface *definition* into this package (or a common package), and then you don't need this import anymore.